git-publish

```diff
@@ -553,9 +553,15 @@ def main():
         tag(tag_name_staging(topic), tag_message, annotate=anno, force=True)
 
     prefix = options.prefix
-    if prefix is None:
+    if prefix is not None:
+        # when prefix == "", it will clear per-branch prefix
```

Comment:
I used empty string to allow user cleanup the subject prefix, like:

  --subject-prefix ""

Maybe we can provide a "--clear-subject-prefix", or... do you have any suggestion on how we should clear an per-branch subject-prefix?

Strip is used to allow user specify something like --subject-prefix " " (with spaces inside subject prefix). But I can remove it for sure.